### ISSUE TITLE
Minor HTML accessibility fixes

### ISSUE THREAD

**Issue by moates** [moates/onlyBans#25]
- [x] The language of the document is not identified or a lang attribute value is invalid.
- [ ] `<h1 id="ui-dialog-title"></h1>` has no content and should be deleted or given
content
- [x] attendWorkshop Change use of heading tags to div class
- [ ]

**Comment by moates** [comment]
The empty header is injected by sugarcube & i don't wanna go that deep into hacking a fix without just adjusting the src code. I filed an issue about it and am going to leave this be for now. 

https://github.com/tmedwards/sugarcube-2/issues/212

**Comment by moates** [comment]
update that Tmedwards pointed out the <h1> div is always set to display: none or something similar if it's not in view, which means no screenreader should be picking it up. 

So either (1) auditors were relying on a meh automated tool or (2) there's some cases wheter the display param isn't set correctly. keep an eye out for #2, but I think #1 is more likely 

https://github.com/tmedwards/sugarcube-2/issues/212
